docs/content/quickstart/build-real-time-data-lake-tutorial.md

```diff
@@ -0,0 +1,269 @@
+      - shared-tmpfs:/tmp/iceberg
+  mysql:
+    image: debezium/example-mysql:1.1
+    ports:
+      - "3306:3306"
+    environment:
+      - MYSQL_ROOT_PASSWORD=123456
+      - MYSQL_USER=mysqluser
+      - MYSQL_PASSWORD=mysqlpw
+
+volumes:
+  shared-tmpfs:
+    driver: local
+    driver_opts:
+      type: "tmpfs"
+      device: "tmpfs"
+```
+
+The Docker Compose environment consists of the following containers:
+- SQL-Client: Flink SQL Client, used to submit queries and visualize their results.
+- Flink Cluster: a Flink JobManager and a Flink TaskManager container to execute queries.
+- MySQL: mainly used as a data source to store the sharding table.
+
+***Note:***
+1. To simply this tutorial, the jar packages required has been packaged into the SQL-Client container.
+If you want to run with your own Flink environment, remember to download the following packages and then put them to `FLINK_HOME/lib/`.
+   
+   **Download links are available only for stable releases.**
+   - [flink-sql-connector-mysql-cdc-2.1-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-mysql-cdc/2.1-SNAPSHOT/flink-sql-connector-mysql-cdc-2.1-SNAPSHOT.jar)
+   - [flink-shaded-hadoop-2-uber-2.7.5-10.0.jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop-2-uber/2.7.5-10.0/flink-shaded-hadoop-2-uber-2.7.5-10.0.jar)
+   - [iceberg-flink-1.13-runtime-0.13.0-SNAPSHOT.jar](https://raw.githubusercontent.com/luoyuxia/flink-cdc-tutorial/main/flink-cdc-iceberg-demo/sql-client/lib/iceberg-flink-1.13-runtime-0.13.0-SNAPSHOT.jar)
+   
+   Currently, the Iceberg official `iceberg-flink-runtime` jar that supports Flink 1.13 isn't released. 
+   Here, we provide a `iceberg-flink-runtime` jar supporting Flink 1.13, which is built based on the master branch of Iceberg. 
+   You can download the `iceberg-flink-runtime` jar from the [apache official repository](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-flink-runtime/) once Iceberg 0.13 is released.
+2. All the following commands for entering the container should be executed in the directory of the `docker-compose.yml` file.
+
+To start all containers, run the following command in the directory that contains the `docker-compose.yml` file:
+```shell
+docker-compose up -d
+```
+This command automatically starts all the containers defined in the Docker Compose configuration in a detached mode. Run `docker ps` to check whether these containers are running properly.
+We can also visit [http://localhost:8081/](http://localhost:8081/) to see if Flink is running normally.
+![Flink UI](/_static/fig/real-time-data-lake-tutorial/flink-ui.png "Flink UI")
+
+### Preparing data in databases
+1. Enter mysql's container:
+    ```shell
+    docker-compose exec mysql mysql -uroot -p123456
+    ```
+2. Create databases/tables and populate data:
+
+   Create a logical sharding table `user` sharded in different databases and tables physically, and one of them misses a column.
```

Comment:
Can you explain why we need to miss a column for one of the shard tables ?

Reply:
It's mainly used to show that Flink CDC can allow the shard tables' columns missmatach for some times it may happen in user real case.

Reply:
I think we'd better to add sentences to describe this point.

Reply:
I think it's also fine to remove the missing column to avoid confusing. We don't need to show every feature in a demo.